docs/codegen/generate-file.md

```diff
@@ -8,18 +8,28 @@ blocks in [Terramate configuration files](../config-overview.md).
 
 Each `generate_file` block requires a single label that is the path
 where the generated file will be saved.
-For more details about how code generation use labels check the [Labels Overview](overview.md#labels)) docs.
+For more details about how code generation use labels check the [Labels Overview](overview.md#labels) docs.
 
-Inside the block, the **`content`** attribute defines the string that will be
-written on the file.
+The block has an optional **`context`** attribute which overrides the [generation context]
+(overview.md#generation-context).
 
-The value of the **`content`** attribute may include:
+The **`content`** attribute defines the string that will be written on the file.
 
-- Terramate Global references `global.*`
-- Terramate Metadata references `terramate.*`
+The value of the **`content`** has access to different Terramate features 
+depending on the `context` defined.
+
+For `context=root` it has access to:
+
+- Terramate Project Metadata references `terramate.root.*` and `terramate.stacks.*`
```

Comment:
Maybe add a link to Terramate Functions, it will not have access to all tm_ functions, there we have the information of which is available etc.

Reply:
I did now but in the line below.